frontend/components/top_nav/SiteTopNav/_styles.scss

```diff
@@ -97,13 +101,20 @@
 
   &__link {
     color: $core-white;
-    padding: 15px 24px;
+    padding: 15px 10px;
     text-decoration: none;
 
-    // Prevents user menu from overlapping with nav bar buttons
-    @media (max-width: 825px) {
+    @media (min-width: 530px) {
+      padding: 15px 16px;
+    }
+
+    @media (min-width: 600px) {
       padding: 15px 18px;
     }
+    
+    @media (min-width: $break-sm) {
+      padding: 15px 24px;
+    }
```

Comment:
Hardcoding the pixel values here because they are specific to the navigation. Adding or removing a navigation item would require adjusting these values. I think that's okay since we very rarely modify the primary navigation.